### DISCUSSION TITLE
Can I use private repositories on a GitHub free account for a commercial project?

### DISCUSSION THREAD

**Discussion by theavijitsinha** [community/community#22337] [New to GitHub]
<p>Hi</p>
<p>Now that unlimited private repositories are available on a GitHub free account, I was wondering if we can use these private repositories for commercial projects. I’ve read the terms of service and I don’t see anything explicitly forbidding it.</p>
<p>Regards</p>
<p>Avijit</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/theavijitsinha">@theavijitsinha</a>,</p>
<p>Thanks for being here, you might use your new private repos for commercial purposes as long as it’s usage remains within our <a href="https://help.github.com/articles/github-terms-of-service/#e-private-repositories" rel="nofollow noopener">terms of service</a>. Keep in mind there are certain features that won’t be available free ie:</p>
<p>Private repositories on <strong>GitHub Pro</strong> also include:</p>
<ul>
<li>Unlimited collaborators</li>
<li>GitHub Pages</li>
<li>Wikis</li>
<li>Protected branches</li>
<li>Code owners</li>
<li>Repository insight graphs: Pulse, Contributors, Traffic, Commits, Code frequency, Network, and Forks</li>
</ul>
<p>I hope this helps, and please let us know if you have any other questions.</p>
